src/main/groovy/com/twcable/gradle/cqpackage/CqPackagePlugin.groovy

```diff
@@ -103,10 +89,35 @@ class CqPackagePlugin implements Plugin<Project> {
     }
 
 
+    @SuppressWarnings("GroovyAssignabilityCheck")
+    private void addDepConfiguration(Project project) {
+        project.logger.debug("Creating configuration: ${CQ_PACKAGE}")
+        def cqPackageConf = project.configurations.create CQ_PACKAGE
+
+        // attach to "runtime", but don't insist that it have to be there first (or will ever be there)
+        def runtimeConf = project.configurations.findByName("runtime")
```

Comment:
It will execute the closure after the Java Plugin is applied, ensuring the runtime configuration is available.